Source/Resource/Plugin/Language/fr.csv

```diff
@@ -414,80 +414,80 @@ File size;Taille du fichier
 Folder (R);Dossier (R)
-Folder (R)_Hint;Sélectionne un dossier à étudier (avec tous ses sous-dossiers)
-Folder (Recursively);Dossier (Récursif)
-Folder_Hint;Sélectionne un dossier à étudier
+Folder (R)_Hint;Sélectionner un dossier à étudier (avec tous ses sous-dossiers)
+Folder (Recursively);Dossier (Récursivement)
+Folder_Hint;Sélectionner un dossier à étudier
 FolderName;Nom du dossier
-Forced;
+Forced;Mode forcé
 Format;Format
 Format_Commercial;Nom commercial
-Format_Commercial_IfAny;Nom commercial
-Format_Compression;
-Format_Description;Description du format
-Format_Info;Infos sur le format
-Format_Level;
-Format_Profile;Profil du format
-Format_Settings;Paramètres du format
-Format_Settings_BVOP;Paramètres du format, BVOP
-Format_Settings_CABAC;Paramètres du format, CABAC
-Format_Settings_Emphasis;Emphase
-Format_Settings_Endianness;Paramètres du format, Endianness
-Format_Settings_Firm;Paramètres du format, Entreprise
-Format_Settings_Floor;Paramètres du format, Seuil
-Format_Settings_FrameMode;Type de répétition
-Format_Settings_GMC;Paramètres du format, GMC
-Format_Settings_GOP;Paramètres du format, GOP
-Format_Settings_ITU;Paramètres du format, ITU
-Format_Settings_Law;Paramètres du format, Law
-Format_Settings_Matrix;Paramètres du format, Matrice
-Format_Settings_Matrix_Custom;Personnalisée
-Format_Settings_Matrix_Default;Par défaut
-Format_Settings_Mode;Paramètres du format, Mode
-Format_Settings_ModeExtension;Extension du mode
-Format_Settings_PacketBitStream;Paramètres du format, Packet bitstream
-Format_Settings_PictureStructure;Paramètres du format, structure de l'image
-Format_Settings_PS;Paramètres du format, PS
-Format_Settings_Pulldown;Paramètres du format, Pulldown
-Format_Settings_QPel;Paramètres du format, QPel
-Format_Settings_RefFrames;Paramètres du format, RefFrames
-Format_Settings_SBR;Paramètres du format, SBR
-Format_Settings_Sign;Paramètres du format, Signe
-Format_Settings_SliceCount;
-Format_Settings_Wrapping;Paramètres du format, mode d'empaquetage
-Format_Tier;
-Format_Url;Lien Internet sur le format
+Format_Commercial_IfAny;Nom commercial IfAny
+Format_Compression;Compression
+Format_Description;Description
+Format_Info;Infos
+Format_Level;Niveau
+Format_Profile;Profil
+Format_Settings;Paramètres
+Format_Settings_BVOP; BVOP
+Format_Settings_CABAC; CABAC
+Format_Settings_Emphasis; Emphase
+Format_Settings_Endianness; Boutisme
+Format_Settings_Firm; Entreprise
+Format_Settings_Floor; Seuil
+Format_Settings_FrameMode; Répétition
+Format_Settings_GMC; GMC
+Format_Settings_GOP; GOP
+Format_Settings_ITU; ITU
+Format_Settings_Law; Law
+Format_Settings_Matrix; Matrice
+Format_Settings_Matrix_Custom; Personnalisé
+Format_Settings_Matrix_Default; Valeur par défaut
+Format_Settings_Mode; Mode
+Format_Settings_ModeExtension; Extension
+Format_Settings_PacketBitStream; Packet bitstream
+Format_Settings_PictureStructure; Structure d'image
+Format_Settings_PS; PS
+Format_Settings_Pulldown; Pulldown (vers le bas)

```

Comment:
I prefer no try to translate.
https://fr.wikipedia.org/wiki/T%C3%A9l%C3%A9cin%C3%A9ma#Le_3:2_pulldown